src/content/en/fundamentals/performance/webpack/decrease-frontend-size.md

```diff
@@ -13,7 +13,27 @@ description: How to use webpack to make your app as small as possible
 One of the first things to do when you’re optimizing an application is to make it as small as
 possible. Here’s how to do this with webpack.
 
-## Enable minification
+## Use the production mode (webpack 4 only) {: #use-the-production-mode }
+
+Webpack 4 introduced the new `mode` flag. You could set this flag to `'development'` or
+`'production'` to hint webpack that you’re building the application for a specific environment:
+
+    // webpack.config.js
+    module.exports = {
+      mode: 'production',
+    };
+
+Make sure to enable the `production` mode when you’re building your app for production.
+This will make webpack apply optimizations like minification, removal of development-only code
+in libraries, [and more](https://medium.com/webpack/webpack-4-mode-and-optimization-5423a6bc597a).
+
+### Further reading {: .hide-from-toc }
+
+* [What specific things the `mode` flag configures](https://medium.com/webpack/webpack-4-mode-and-optimization-5423a6bc597a)
+
+## Enable minification {: #enable-minification }
+
+Note: if you’re using [webpack 4 with the production mode](#enable-the-production-mode), the bundle-level minification is already enabled. You’ll only need to enable loader-specific options.
```

Comment:
I like that we're still including the webpack 3 guidance but clearly calling out where 4 has guidance that supersedes it like this.